guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1109 @@
+# Updating from v12 to v13
+
+## Before you start
+
+v13 requires Node 14.x or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
+
+Once you've got Node up-to-date, you can install v13 by running `npm install discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/opus` for voice support.
```

Comment:
Not quite sure about this so I have a question, the latter part of this guide says users need to use Discord.js' voice library in order to get voice support? If this is so, then this Opus bindings library should be another implementation or this is a combination of both the voice and this Opus library, then? If it's otherwise, I've included a suggestion.
```suggestion
Once you've got Node up-to-date, you can install v13 by running `npm install discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/voice` for voice support.
```

Reply:
> Not quite sure about this so I have a question, the latter part of this guide says users need to use Discord.js' voice library in order to get voice support? If this is so, then this Opus bindings library should be another implementation or this is a combination of both the voice and this Opus library, then? If it's otherwise, I've included a suggestion.

Voice depends on an Opus library with bindings for Node.js. If that's so, maybe I should just change the statement to `npm install discord.js @discordjs/opus @discordjs/voice` for a complete installation setup (assuming Discord.js' Opus library is the recommended one). Or perhaps either the above should stay as-is, or don't edit the above statement at all?